styles/prosilver/template/ideas.js

```diff
@@ -260,9 +260,11 @@
 			$.get(url, {duplicate: Number(value)}, function(res) {
 				if (res) {
 					if (value) {
-						$obj.duplicateLink.text('idea.php?id=' + value)
-							.attr('href', 'idea.php?id=' + value)
-							.show();
+						$obj.duplicateLink.text(function() {
+							return $(this).attr('data-l-msg');
+						})
+						.attr('href', '../app.php/idea/' + value)
```

Comment:
app.php is always there... the rewriting a syou know just affects if you try access the page without app.php in the url